jokes.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+func getDadJoke() (string, error) {
+	u := "https://icanhazdadjoke.com/"
+	resp, err := http.Get(u)
```

Comment:
i might have to new up a client and do it the long way because of this header

https://github.com/dhruv11/rudolph/pull/3/files#diff-7ddfb3e035b42cd70649cc33393fe32cL147

Reply:
Yep, I wondered if that was needed ... the change you've made is good.